docs/site/BelongsTo-relation.md

```diff
@@ -75,18 +98,44 @@ export interface OrderRelations {
 export type OrderWithRelations = Order & OrderRelations;
 ```
 
-The definition of the `belongsTo` relation is inferred by using the `@belongsTo`
-decorator. The decorator takes in a function resolving the target model class
-constructor and designates the relation type. It also calls `property()` to
-ensure that the decorated property is correctly defined.
+The above example can also be expanded with explicit relation and property
+definitions as follows,
+
+- The standard naming convention for the foreign key property in the source
+  model is `relation name` + `Id`. (e.g. Order.customerId).
+- If the property name in the source model has to be customized then the `name`
+  attribute in the relation definition (the relation name) has to be stated
+  explicitly. Also, the corresponding `hasMany` relation in the `target` model
+  has to be modified with the correct `keyTo` value.
+- In the following example :
+  - a relation definition is added to the decorator of the property with a
+    customized name (`cust_Id`).
+  - a property definition is added with the customized db column name
+    (`customer_id`).
+
+```ts
+class Order extends Entity {
+  // constructor, properties, etc.
+  @belongsTo(
+    () => Customer,
+    {keyFrom: 'customerId', keyTo: 'id', name: 'customer'},
```

Comment:
`keyFrom: 'customerId' `-> `keyFrom: 'cust_Id'` ?

Reply:
keyFrom is dummy anyway

Reply:
agreed

Reply:
I don't know. It would be really confusing if I didn't know LB4 that much. 
If `keyFrom` is always set to the decorated property name, why not set it to `cust_id` or even remove it?

or you can add a simplified one after it as:

``` 
or we can simplified it as 
@belongsTo(
    () => Customer,
    {name: 'customer'},
    {name: 'different_property_name_for_db'} // pass in the third param if you'd like to use different name for db column 
)
my_customer_id: number;
```

Personally I'd like to make docs easy to understand at the first glance instead of having everything up all at once. Like the `getting started` part of LB4, it is simple and also has enough material to start. It's good to have the explanation about using different name for database, appreciate that. So I would probably to break it down as: 
```
here's an example of having customized name
...
you can simplify the above example to ...
...
if you would need to use names of db columns, you can do
...
```

Reply:
The idea is to explain the overall syntax to the developer. we should be able to explain in as few examples as possible, instead of providing multiple combinations.

Reply:
Alright, I will respect your decision on this.

Reply:
@agnes512 I agree with your comment https://github.com/strongloop/loopback-next/pull/3808#discussion_r328731959 , so I have updated the docs as per the suggestions.

Reply:
also comment https://github.com/strongloop/loopback-next/pull/3772#discussion_r326426738 is addressed in this PR .